<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLayerView</class>
 <widget class="QWidget" name="CLayerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>442</width>
    <height>132</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="layer_name"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>–ò–∑–º–µ–Ω–µ–Ω–∏–µ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mode_choose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="placeholderText">
        <string>...</string>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>–ù–µ–∏–∑–º–µ–Ω—è–µ–º—ã–π</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>–ü–µ—Ä–µ–±–æ—Ä –∫–∞—Ä—Ç–∏–Ω–æ–∫</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>–ü–µ—Ä–µ–±–æ—Ä —Ü–≤–µ—Ç–æ–≤</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="settings">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>‚öô</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>–ú–∞—Å–∫–∞:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_mask">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="one_file_">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>1 –∫–∞—Ä—Ç–∏–Ω–∫–∞</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>–ü—É—Ç—å:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path_edit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="path_choose">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>üìÇ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
